snomed/com.b2international.snowowl.snomed.core.rest/src/com/b2international/snowowl/snomed/core/rest/domain/SnomedRf2ExportConfiguration.java

```diff
@@ -57,11 +56,8 @@
 	@ApiParam(value = "The RF2 RefSet file layout to use. Defaults to server configuration key 'snomed.export.refSetLayout'.", allowableValues = "combined,individual")
 	private String refSetLayout;
 	
-	@ApiParam(value = "The maintainerType to use un the release archive name")
-	private Rf2MaintainerType maintainerType = Rf2MaintainerType.SNOMED_INTERNATIONAL;
-	
-	@ApiParam(value = "The nrcCountryCode to use un the release archive name")
-	private String nrcCountryCode = "";
+	@ApiParam(value = "The CodeSystem's id on wich the export was initiated.")
+	private String codeSystemId;
```

Comment:
But this is already specified via the path parameter, why do you need this? Also, the CodeSystem can be calculated from the current branch path on the fly, so feel free to remove this.